Library/Formula/docker-machine-driver-xhyve.rb

```diff
@@ -0,0 +1,34 @@
+require "language/go"
+
+class DockerMachineDriverXhyve < Formula
+  desc "Docker Machine driver for xhyve"
+  homepage "https://github.com/zchee/docker-machine-driver-xhyve"
+  url "https://github.com/zchee/docker-machine-driver-xhyve/archive/v0.2.1.tar.gz"
+  sha256 "11ed10c4ea45249e06c9efad0abf1c91b0ec9c4c2a0818ade78f19f41a17794c"
+
+  head "https://github.com/zchee/docker-machine-driver-xhyve.git"
+
+  depends_on :macos => :yosemite
+  depends_on "go" => :build
+  depends_on "docker-machine"
+
+  def install
+    contents = Dir["{*,.git,.gitignore}"]
+    gopath = buildpath/"gopath"
+    (gopath/"src/github.com/zchee/docker-machine-driver-xhyve").install contents
+
+    ENV["GOPATH"] = gopath
+    ENV["GO15VENDOREXPERIMENT"] = "1"
+
+    cd gopath/"src/github.com/zchee/docker-machine-driver-xhyve" do
+      if build.head?
+        git_commithash = `git rev-parse --short HEAD 2>/dev/null`.delete!("\n")
+      end
+      system "go", "build", "-o", bin/"docker-machine-driver-xhyve", "-ldflags", "'-w -s'", "-ldflags", "-X 'github.com/zchee/docker-machine-driver-xhyve/xhyve.GitCommit=#{git_commithash}+Homebrew'", "./main.go"
```

Comment:
@mikemcquaid Hmm...
If the user builds without `Homebrew`, that value will be commit hash only.
e.g.

```bash
===== Docker Machine xhyve Driver Version 0.2.1 (896d3f5) =====
```

I think beautifully also compatible build without `Homebrew` is `Homebrew`, `#{git_hash}+Homebrew`.

If I want to parse the version for some reason, I can determine whether build with `Homebrew` only regexp of a backward match($).
If `Homebrew HEAD-#{git_mash}`, Parse of version will be complicated.

What do you think?